import_ldraw.py

```diff
@@ -204,7 +211,7 @@ def parse_line(self, line):
         color = line[1]
 
         if color == '16':
-            color = self.colour
+            color = self.color
```

Comment:
Let me clarify: all new code should use American spelling. Yes, some existing code does not, but in due time that would be remedied. This PR's scope is adding a linked bricks feature, not cleaning up the variable names. :) However, I'll let this pass because it's your first PR and reverting it back would a lot of work. Make more sense?